flink-connectors/flink-connector-rabbitmq/src/test/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSourceTest.java

```diff
@@ -382,6 +414,27 @@ public boolean isEndOfStream(String nextElement) {
 		}
 	}
 
+	private class CustomDeserializationSchema implements RMQDeserializationSchema<String> {
+		@Override
+		public TypeInformation<String> getProducedType() {
+			return TypeExtractor.getForClass(String.class);
+		}
+
+		@Override
+		public void processMessage(Envelope envelope, AMQP.BasicProperties properties, byte[] body, RMQCollector collector) throws IOException {
+			List<String> messages = new ArrayList();
+			messages.add("I Love Turtles");
+			messages.add("Brush your teeth");
```

Comment:
:paintbrush:  :grimacing:

Reply:
i am sorry i am known to play around with tests strings :D i can change it ... but then what do you have against teeth brushing !? explain yourself mister ! :D

Reply:
Ha! I really like your test data, fun data over everything!